From bfcbb19c954fa20a68708e707e76c725058e001f Mon Sep 17 00:00:00 2001
From: Shane Loretz <sloretz@osrfoundation.org>
Date: Tue, 1 Mar 2022 16:54:10 -0800
Subject: [PATCH] Install yaml.h to INSTALL_INCLUDE_DIR

Signed-off-by: Shane Loretz <sloretz@osrfoundation.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4f81148..80b12cb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,7 @@ target_include_directories(yaml PUBLIC
 install(
   FILES
     include/yaml.h
-  DESTINATION include COMPONENT Development
+  DESTINATION ${INSTALL_INCLUDE_DIR} COMPONENT Development
   )
 
 install(
-- 
2.25.1

